### PR TITLE
Login api connect

### PR DESCRIPTION
**Pull request by jahzielv**
we can now create new accounts and login as correct user types!

### CODE REVIEW

File: src/components/LoginPage.js [hunk 1]
```diff
@@ -82,10 +82,11 @@ export class LoginPage extends React.Component {
             <Button type="submit" value="login" variant="raised" color="primary">
               Login
             </Button>
-
             {isLoginPending && <Snackbar open={true} autoHideDuration={6000} message={<span>Logging you in...</span>} />}
             {/* TODO: find a way to make this conditional on what type of user you are */}
-            {isLoginSuccess && <Redirect to="/dashboard" />}
+            {isLoginSuccess && userPath != null && <Redirect to={userPath} />}
+            {/* {isLoginSuccess && (userType == "LM") && <Redirect to="/input" />} */}
+            {/* {userType == "QA" && <Redirect to="/dashboard" />} */}
```
Comment (mhelmstadter) [inline src/components/LoginPage.js]: do we still need the commented out code here eventually?

File: src/redux/auth-reducer.js [hunk 2]
```diff
@@ -23,55 +27,89 @@ function setLoginError(loginError) {
...
+}
+
+function callLoginApi(ID, password, callback) {
+  // setTimeout(() => {
+  //   if (email === 'admin@example.com' && password === 'admin') {
+  //     return callback(null);
+  //   } else {
+  //     return callback(new Error('Invalid email and password'));
+  //   }
+  // }, 1000);
```
Comment (mhelmstadter) [inline src/redux/auth-reducer.js]: Will we need this commented out setTimeout function now that the actual login logic has been implemented?

File: src/redux/auth-reducer.js [hunk 3]
```diff
@@ -23,55 +27,89 @@ function setLoginError(loginError) {
...
+  async function f(idnum) {
```
Comment (mhelmstadter) [inline src/redux/auth-reducer.js]: Can we rename this function?

File: src/components/LoginPage.js (hunk 1 above)
Comment (jahzielv) [reply src/components/LoginPage.js]: Nah we should be fine without them, just forgot to delete them.

File: src/redux/auth-reducer.js (hunk 2 above)
Comment (jahzielv) [reply src/redux/auth-reducer.js]: Nope, that can get tossed out too

File: src/redux/auth-reducer.js (hunk 3 above)
Comment (jahzielv) [reply src/redux/auth-reducer.js]: No.

Comment (jahzielv) [reply src/redux/auth-reducer.js]: LOL jk yes pls let's rename it 😂 😂

**Review by mhelmstadter — APPROVED** [review]
LGTM

**Pull request merged**